server/models/autoIncrementModel.js

```diff
@@ -0,0 +1,27 @@
+import mongoose from 'mongoose';
+
+const CounterSchema = new mongoose.Schema({
```

Comment:
What is the purpose of this Model?

Reply:
**Also, do we really need it?**
Or we can rely on mongo unique ObjectId.

Because of that, it'll have access 1 more time to DB.